stairctl/gradle.lockfile

```diff
@@ -1,140 +0,0 @@
-# This is a Gradle generated file for dependency locking.
```

Comment:
not that I"m complaining but why the removal of hte .lockfiles?

Reply:
We've been removing lockfiles in all of our repos to make dependency upgrades easier.